packages/fxa-auth-server/lib/senders/emails/partials/changePassword/index.txt

```diff
@@ -0,0 +1 @@
+change-password-plaintext = "If you suspect that someone is trying to gain access to your account, please change your password. { $passwordChangeLink }"
```

Comment:
Ahhh, cool - I also added this in #10418 (but I didn't do a partial like you did!)

This appears to be a variation on the `automatedEmailChangePassword` partial, which is itself a variation on `automatedEmailResetPassword` and `automatedEmailNoAction`. I think it would be worth a follow up to try to come up with a dynamic partial that could handle all these cases and allow us to remove all these partials.

What you think?

Reply:
I did find it kind of weird that the plaintext version of this differs from what the HTML email reads, it makes me think the HTML was changed sometime for one of these template and the corresponding TXT was not updated. I kind of wondered if we had a meeting with... not sure, maybe Design? if we could just make this more consistent across the board rather than making a more dynamic partial. Either way I can file an issue for it.